.dagger-ci/daggerci/requirements.txt

```diff
@@ -0,0 +1,12 @@
+anyio>=3.7.1
+dagger>=1.3.0
```

Comment:
Is this actually correct here?

Reply:
Yeah, I think so. It works in the job `Run pytest`.

Reply:
`dagger` as such maybe should not be part of `requirements.txt` because it is not python thing. `dagger-io` is just python bindings for `dagger`.